backend/controller/sql/types.go

```diff
@@ -1,61 +1,10 @@
 package sql
 
 import (
-	"database/sql/driver"
-	"fmt"
 	"time"
 
 	"github.com/alecthomas/types/optional"
-	"github.com/google/uuid"
-	"github.com/oklog/ulid/v2"
 )
 
-type NullKey = optional.Option[Key]
```

Comment:
We have no more uuid/Key columns so this code was all being unused. Is it ok to remove it like I have or are we wanting to keep it for other UUID stuff coming down the road?

Reply:
Nope nuke it. We can always restore it from history if we need it.